File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/")

    print(driver.title)

    driver.find_element(By.XPATH, "//a[@id='about-link']").click()

    print("About page title is: ", driver.title)

    #driver.close()

    driver.get("https://v1.training-support.net/selenium/login-form")

    print(driver.title)

    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")
    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")
    driver.find_element(By.XPATH,"//button[text()='Log in']").click()

    #Verifying login
    message = driver.find_element(By.XPATH,"//div[@id='action-confirmation']")
    print("Login message: ",message.text)

    driver.close()

File: Selenium/Activities/Activity3.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {

	public static void main(String[] args) {
		
		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/");
		
		System.out.println(driver.getTitle());
		
		driver.findElement(By.xpath("//a[@id='about-link']")).click();
	    
	    System.out.println("About page: " + driver.getTitle());
	    
	    //driver.close();
	    
	    driver.get("https://v1.training-support.net/selenium/login-form");
		
		System.out.println(driver.getTitle());
		
		driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
		driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
		driver.findElement(By.xpath("//button[text()='Log in']")).click();
		
		//Verifying login 
		String message = driver.findElement(By.xpath("//div[@id='action-confirmation']")).getText();
		System.out.println("Login message: " + message);
		
		driver.close();

	}

}
